ext/openssl/openssl.stub.php

```diff
@@ -22,3 +22,49 @@ function openssl_x509_read($cert) {}
 
 /** @return false|void */
 function openssl_x509_free($x509) {}
+
+function openssl_pkcs12_export_to_file($x509cert, string $filename, $priv_key, string $pass, array $args = UNKNOWN): bool {}
+
+function openssl_pkcs12_export($x509 , &$out, $priv_key, string $pass, array $args = UNKNOWN): bool {}
+
+function openssl_pkcs12_read(string $pkcs12, &$certs, string $pass): bool {}
+
+function openssl_csr_export_to_file($csr, string $outfilename, bool $notext = true): bool {}
+
+function openssl_csr_export($csr, &$out, bool $notext = true): bool {}
+
+/** @return resource|false */
+function openssl_csr_sign($csr, $cacert, $priv_key, int $days, array $config_args = UNKNOWN, int $serial = 0) {}
+
+/** @return resource|false */
+function openssl_csr_new($dn, &$privkey, array $configargs = UNKNOWN, array $extraattribs = UNKNOWN) {}
+
+/** @return array|false */
+function openssl_csr_get_subject($csr, bool $use_shortnames = true) {}
+
+/** @return resource|false */
+function openssl_csr_get_public_key($csr, bool $use_shortnames = true) {}
+
+/** @return resource|false */
+function openssl_pkey_new(array $configargs = UNKNOWN) {}
+
+function openssl_pkey_export_to_file($key, string $outfilename, string $passphrase = UNKNOWN, array $configargs = UNKNOWN): bool {}
+
+function openssl_pkey_export($key, &$out, string $passphrase = UNKNOWN, array $configargs = UNKNOWN): bool {}
+
+/** @return resource|false */
+function openssl_pkey_get_public($cert) {}
+
+function openssl_pkey_free($key) {}
```

Comment:
We can also specify the `void` return here, so:
```suggestion
/** @param resource $key */
function openssl_pkey_free($key): void {}
```